src/service-command.ts

```diff
@@ -25,4 +32,18 @@ export default abstract class extends Command {
   status(s: number) {
     return ['unknown', 'stopped', 'starting', 'partial', 'running'][s]
   }
+
+  async getAuthorizedServiceInfo(id: string, versionHash: string): ServiceInfo {
+    const list = await this.executeAndCaptureError(services.account.id, services.account.tasks.list)
```

Comment:
We could use the `account:list` command directly instead of calling directly the service with something like that
```
await AccountList.run('-s') // run the command in silent mode
```

Reply:
I'm not sure this is a good practice. maybe we can create some helper/provider classes like in the Go cli version. need to think about but definitely overall js cli need a bit refactoring